index.bs

```diff
@@ -395,6 +395,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: User consent means the user agrees with what they are being asked, i.e., it encompasses reading and understanding prompts.
     An [=authorization gesture=] is a [=ceremony=] component often employed to indicate [=user consent=].
 
+: <dfn>User Handle</dfn>
+:: The user handle is specified by a [=[RP]=] and is a unique identifier for a user account with that [=[RP]=]. A user handle is
+    an opaque byte array with a maximum size of 64 bytes.
```

Comment:
One other thing: could you use [`[=byte sequence=]`](https://infra.spec.whatwg.org/#byte-sequence) instead of "byte array"?

Reply:
Absolutely, should we do that in [§5.1](https://w3c.github.io/webauthn/#sec-authenticator-data) as well?

Reply:
Yes, but probably in a separate PR to avoid scope creep. :)

Reply:
My thought as well. :)